lib/internal/process.js

```diff
@@ -65,6 +65,13 @@ function setupConfig(_source) {
     if (value === 'false') return false;
     return value;
   });
+  if ( process.config.variables.v8_enable_i18n_support && global.hasOwnProperty('Intl') 
+        && Intl.hasOwnProperty('v8BreakIterator') && process.config.variables.icu_small 
+        && !process.icu_data_dir ) {
+      Intl.v8BreakIterator = function v8BreakIterator() { 
+        throw Error('Data not available...'); 
+    }
+  }
```

Comment:
Can you `delete process.icu_data_dir` afterwards?  It shouldn't leak out into user code.